specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
We should limit what we index on to something that contains just the essential key they're using: "TabulatorNum", "BatchId", "RecordId". We should just drop stuff that shouldn't be in there anyway for CORA reasons: _These columns seem problematic: "ImprintedId", "CountingGroup", "PrecinctPortion".  And in this case the "BallotType" (ballot style) seems to include a precinct number, which is also a problem given the way we mix ballots up in batches in Colorado._ The CvrNumber is redundant, just seems to be another unique incrementing recordid.